### PR TITLE
feat(scope): add hybrid example of both initialize scoped atom on render and nested scopes

### PR DESCRIPTION
**Pull request by yf-yang**

### CODE REVIEW

**Comment by dai-shi** [comment]
It should be nice to have such an example. Please create it as a new example.

**Comment by yf-yang** [comment]
Ah, sorry, I thought I was placing it in a new folder

File: examples/03_hybrid/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        <Counter1 />
+      </ScopeProvider>
+      <h1>Second Provider</h1>
+      <ScopeProvider atoms={[scopedAtom1]}>
+        <Counter1 />
+      </ScopeProvider>
+      <h1>Provider with initial value 10</h1>
+      <ScopeProviderWithInitializer
+        atomValues={
+          [[scopedAtom1, 10]] as unknown as Parameters<
```
Comment (yf-yang) [inline examples/03_hybrid/src/App.tsx]: Here, the typescript signature is broken. Do you know how to fix it?

Comment (yf-yang) [reply examples/03_hybrid/src/App.tsx]: Tried to copy `InferAtoms` from `useHydrateAtoms` and it worked. If that type could be exported, it could fix it.

Comment (dai-shi) [reply examples/03_hybrid/src/App.tsx]: Just copying it is fine for now. We may change something in the future.

**Comment by dai-shi** [comment]
https://codesandbox.io/s/github/yf-yang/jotai-scope/tree/feat/example/examples/03_hybrid
☝️ works fine.

Please follow eslint rules.

**Review by dai-shi — APPROVED** [review]
Thanks!

**Comment by dai-shi** [comment]
On second thought, I wouldn't hesitate too much about providing `ScopeProviderWithInitialValues` from `jotai-scope`.

**Pull request merged**